Gems/PhysX/Code/Source/FixedJointComponent.cpp

```diff
@@ -91,5 +92,81 @@ namespace PhysX
                 leadFollowerInfo.m_followerBody->m_bodyHandle);
             m_jointSceneOwner = leadFollowerInfo.m_followerBody->m_sceneOwner;
         }
+        m_nativeJoint = FixedJointComponent::GetNativeJoint();
+        if (m_nativeJoint)
+        {
+            JointRequestBus::Handler::BusConnect(AZ::EntityComponentIdPair(GetEntityId(), GetId()));
+        }
+    }
+
+    void FixedJointComponent::DeinitNativeJoint()
+    {
+        JointRequestBus::Handler::BusDisconnect();
+        m_nativeJoint = nullptr;
     }
+
+    physx::PxFixedJoint* FixedJointComponent::GetNativeJoint() const
+    {
+        auto* sceneInterface = AZ::Interface<AzPhysics::SceneInterface>::Get();
+        AZ_Assert(sceneInterface, "No sceneInterface");
+        const auto* joint = sceneInterface->GetJointFromHandle(m_jointSceneOwner, m_jointHandle);
+        AZ_Assert(joint->GetNativeType() == NativeTypeIdentifiers::FixedJoint, "It is not PhysxFixed Joint");
+        physx::PxJoint* native = static_cast<physx::PxJoint*>(joint->GetNativePointer());
+        physx::PxFixedJoint* nativeFixed = native->is<physx::PxFixedJoint>();
+        return nativeFixed;
+    }
+
+
+    float FixedJointComponent::GetPosition() const
+    {
+        AZ_Warning("FixedJointComponent::GetPosition", false, "Cannot get position in fixed joint");
+        return 0;
+    }
+
+    float FixedJointComponent::GetVelocity() const
+    {
+        AZ_Warning("FixedJointComponent::GetVelocity", false, "Cannot get velocity in fixed joint");
+        return 0;
+    }
+
+    AZ::Transform FixedJointComponent::GetTransform() const
+    {
+        const auto worldFromLocal = m_nativeJoint->getRelativeTransform();
+        return AZ::Transform(
+            AZ::Vector3{ worldFromLocal.p.x, worldFromLocal.p.y, worldFromLocal.p.z },
+            AZ::Quaternion{ worldFromLocal.q.x, worldFromLocal.q.y, worldFromLocal.q.z, worldFromLocal.q.w },
+            1.f);
+    }
```

Comment:
There are helpers to convert from px types to az types and viceversa, this function can be:

````
    AZ::Transform FixedJointComponent::GetTransform() const
    {
        return PxMathConvert(m_nativeJoint->getRelativeTransform());
    }
````